### PR TITLE
Add VIAaaS egg and close #1207

### LINKED ISSUE #1207: ViaVersion as a Service
Please fill out the information bellow and remove from the line up
Please understand how Pterodactyl works when you are requesting an egg. (ie. docker-compose doesn't work for a pterodactyl server)
---------------

Service: (Ex. minecraft/factorio/etc)

ViaVersion as a Service (VIAaaS)

Does this expand an already existing service: Y/N

No.

Link to game: (Ex. minecraft.net/factorio.com/etc)

Not sure what I should put here... https://minecraft.net/ I suppose?

Links for server downloads: This needs to be an official link and not one that is hosted on some forum page or a personal github page.

https://github.com/ViaVersion/VIAaaS

Links for install steps/docs:

https://github.com/ViaVersion/VIAaaS#setting-up-server-instance

~~Sorry about that last PR, this time I'm sure that an egg does not already exist.~~

### PR DESCRIPTION
**Pull request by regulad**
Closes #1207 

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

* [x] Does your submission pass tests (server is connectable)?
* [ ] Does your server use a custom docker image?
* [x] Have you added the server to the main README.md?
* [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

**Comment by koutsie** [comment]
LGTM!

**Review by koutsie — APPROVED** [review]

File: minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    },
+    "variables": [
+        {
+            "name": "VIAaaS Jar File",
+            "description": "The name of the jarfile to be used when downloading & running VIAaaS.",
+            "env_variable": "SERVER_JARFILE",
+            "default_value": "VIAaaS-all.jar",
+            "user_viewable": true,
+            "user_editable": true,
```
Comment (DatMayo) [inline minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json:37]: Is there a reason, why you've allow the user to change the binary name?

File: minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-06-28T11:55:37-04:00",
+    "name": "VIAaaS",
+    "author": "regulad@outlook.com",
+    "description": "VIAaaS - ViaVersion as a Service - Standalone ViaVersion proxy",
+    "features": null,
+    "images": [
+        "ghcr.io\/pterodactyl\/yolks:java_16"
```
Comment (DatMayo) [inline minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json:13]: Why not use one of the existing images?

Comment (Software-Noob) [reply minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json:13]: It's official pterodactyl image?

**Comment by Software-Noob** [comment]
Can you please link their Github repository in the readme as it contains useful information to get this working for older versions

**Comment by regulad** [comment]
Yes, give me one second.

> 
> 
> Can you please link their Github repository in the readme as it contains useful information to get this working for older versions

File: minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json (hunk 1 above)
Comment (regulad) [reply minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json:37]: Most other Minecraft eggs allow changing the binary. If the user wants to update it themselves, for example.

File: README.md [hunk 3]
```diff
@@ -141,6 +141,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
     * [TyphoonLimbo](/minecraft/proxy/java/typhoonlimbo)
     * [Velocity](/minecraft/proxy/java/velocity)
     * [Waterfall](/minecraft/proxy/java/waterfall)
+    * [VIAaaS](/minecraft/proxy/java/viaaas)
```
Comment (Software-Noob) [inline README.md:144]: Please keep the alphabetical order by putting this above waterfall

**Review by Software-Noob — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (regulad) [reply README.md:144]: Ah, ok. Didn't think about that.

File: minecraft/proxy/java/viaaas/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# VIAaaS
+VIAaaS is the Minecraft plugin VIAaaS made standalone as a proxy.
```
Comment (creeper123123321) [inline minecraft/proxy/java/viaaas/README.md:2]: ViaVersion*

**Review by creeper123123321 — CHANGES_REQUESTED** [review]

File: minecraft/proxy/java/viaaas/README.md (hunk 4 above)
Comment (regulad) [reply minecraft/proxy/java/viaaas/README.md:2]: Ah, durr...

**Comment by Software-Noob** [comment]
Some nitpicking but viaas should be between velocity and waterfall in the readme files as in the original commit. The `e` letter is before `i`

**Comment by regulad** [comment]
Rebasing is probably a good idea considering how many README changes have occurred.

**Review by Software-Noob — APPROVED** [review]

**Comment by Software-Noob** [comment]
Thank you for the contribution!

**Pull request merged**